### ISSUE TITLE
Fix button accessibility

### ISSUE THREAD

**Issue by tomascordero** [tomascordero/safety-exit#22]
The button is currently not very accessible. It should meet AA WCAG standards at minimum. The UI should also show a warning if the user selects a font / background color combo that is not accessible. 

[Request URL](https://wordpress.org/support/topic/accessibility-and-wcag2-0-requirements)
